pkg/cli-runtime/wait/wait_test.go

```diff
@@ -191,6 +191,13 @@ func TestUntilDelete(t *testing.T) {
 		name:    "smaller timeout",
 		timeout: time.Millisecond,
 		err:     context.DeadlineExceeded,
+		reactorFunc: func(action clitesting.Action) (bool, runtime.Object, error) {
```

Comment:
this test case should error out when context being done and not invoke get call. This change is altering how `context.DeadlineExceeded` error is generated.
If this test fails in windows, i would reduce the timeout nanosecond and see if that helps so it doesnt trigger the 2nd condition of the ticker

Reply:
changing that to Nano seconds are not working as it is expecting to have the reactorFunction.